repos/spack_repo/builtin/packages/mui/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+
+from spack.package import *
+
+
+class Mui(CMakePackage):
+    """Multiscale Universal Interface: A Concurrent Framework for Coupling Heterogeneous Solvers"""
+
+    homepage = "https://mxui.github.io/"
+    git = "https://github.com/MxUI/MUI.git"
+
+    maintainers("Wendi-L", "SLongshaw")
+
+    license("Apache-2.0", checked_by="blairSmcc03")
+
+    version("2.0", branch="master")
```

Comment:
Branches are not trusted downloads. The repository has a tagged release that can be used. You can add (between lines 13 and 14):

```
    url = "https://github.com/MxUI/MUI/archive/refs/tags/2.0.tar.gz"
```

and change this to use the `sha256`:


```suggestion
    version("2.0", sha256="fdddd4ffe72c22356eb53707567622a9bfb8d17836a9677a980f035e87e1b295")
```

Reply:
Is it okay to have tagged release and git branch? The tagged release is reasonably out of date.

Reply:
Certainly. You can add a master version that relies on the master branch.